src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -42,7 +42,7 @@ class AnalyticsConstants {
         return {
             git {
                 remote {
-                    url(allVars.get('SECURE_REPO_URL'))
+                    url('$SECURE_REPO')
```

Comment:
This was previously ignoring the SECURE_REPO parameter which could have caused confusion.

Reply:
Looks like a good catch.